i18next-parser.config.js

```diff
@@ -0,0 +1,76 @@
+/*eslint no-undef: "error"*/
+/*eslint-env node*/
+
+module.exports = {
+    contextSeparator: '_',
+    // Key separator used in your translation keys
+  
+    createOldCatalogs: true,
+    // Save the \_old files
+  
+    defaultNamespace: 'public',
+    // Default namespace used in your i18next config
+  
+    defaultValue: 'temp',
+    // Default value to give to empty keys
+  
+    indentation: 2,
+    // Indentation of the catalog files
+  
+    keepRemoved: false,
+    // Keep keys from the catalog that are no longer in code
+  
+    keySeparator: '.',
+    // Key separator used in your translation keys
+    // If you want to use plain english keys, separators such as `.` and `:` will conflict. You might want to set `keySeparator: false` and `namespaceSeparator: false`. That way, `t('Status: Loading...')` will not think that there are a namespace and three separator dots for instance.
+  
+    // see below for more details
+    lexers: {
+      hbs: ['HandlebarsLexer'],
+      handlebars: ['HandlebarsLexer'],
+  
+      htm: ['HTMLLexer'],
+      html: ['HTMLLexer'],
+  
+      mjs: ['JavascriptLexer'],
+      js: ['JavascriptLexer'], // if you're writing jsx inside .js files, change this to JsxLexer
+      ts: ['JavascriptLexer'],
+      jsx: ['JsxLexer'],
+      tsx: ['JsxLexer'],
+  
+      default: ['JavascriptLexer'],
+    },
+  
+    lineEnding: 'auto',
+    // Control the line ending. See options at https://github.com/ryanve/eol
+  
+    locales: ['en', 'zh'],
```

Comment:
> Sure, running yarn localize does regenerate them by the way. For now since the translations aren't a big part of the project, I think it doesn't need to be added to the CI yet.

Maybe we could comment out `zh` here to that `yarn localize` won't add them in?